src/commonMain/kotlin/fr/acinq/lightning/bin/Api.kt

```diff
@@ -402,13 +403,40 @@ class Api(
                             is Either.Left -> call.respondText(res.value.message.toString())
                         }
                     }
+                    post("bumpfee") {
+                        val formParameters = call.receiveParameters()
+                        val targetFeerate = FeeratePerKw(FeeratePerByte(formParameters.getLong("feerateSatByte").sat))
+                        val channel = peer.channels.values.filterIsInstance<Normal>().firstOrNull()
+                        val cpfpFeerate = channel?.let { peer.estimateFeeForSpliceCpfp(it.channelId, targetFeerate)?.first }
+                        if (cpfpFeerate == null) {
+                            call.respondText("no channel available")
+                            return@post
+                        }
+                        if (cpfpFeerate == targetFeerate) {
+                            call.respondText("cpfp is useless")
```

Comment:
What if `cpfpFeerate` < `targetFeerate` ?

Also the message is not clear. How about `effective feerate is too low`?

Reply:
The cpfp is useless if the funding tx is already confirmed, or if the requested feerate is equal or below the current feerate. 

But `lightning-kmp` doesn't return the detail, just:
```kotlin
     *         NB: if the output feerate is equal to the input feerate then the cpfp is useless and
     *         should not be attempted.
     */
    suspend fun estimateFeeForSpliceCpfp(channelId: ByteVector32, targetFeerate: FeeratePerKw): Pair<FeeratePerKw, ChannelManagementFees>? {
```

Maybe:
```suggestion
                            call.respondText("either the tx is already confirmed or has a higher feerate")
```